server/apps/publish/archive_publish.py

```diff
@@ -79,8 +79,10 @@ def update(self, id, updates, original):
             user = get_user()
             push_notification('item:publish', item=str(item.get('_id')), user=str(user))
             return item
-        except KeyError:
-            raise SuperdeskApiError.badRequestError(message="A non-existent content id is requested to publish")
+        except KeyError as e:
+            raise SuperdeskApiError.badRequestError(
+                message="Key is missing on article to be published: {}"
+                .format(str(e)))
```

Comment:
I observed this key problem in destinations branch few times, so some of the articles are missing a key but I don't know which one. That's why I am interested in the original KeyError as I need to see what key is creating problem. 

Please let me know if there's a better way of doing that. Cheers.